homeassistant/components/synologydsm/sensor.py

```diff
@@ -162,7 +168,7 @@ def __init__(self, host, port, username, password, temp_unit, use_ssl):
         self.utilisation = self._api.utilisation
         self.storage = self._api.storage
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
+    @Throttle(SCAN_INTERVAL)
```

Comment:
Please revert this change. The user can already change the scan interval via config options. Here we set the minimum interval that the user can not override.